app/models/notification.rb

```diff
@@ -0,0 +1,39 @@
+class Notification < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :discussion
+
+  validates :text, :url, :user, presence: true
+
+  after_create :send_email
+
+  def self.new_discussion(discussion)
+    create(
+      discussion_id: discussion.id,
+      text: I18n.t('notifications.new_discussion', author: discussion.author, discussion: discussion.title),
+      url: root_url(anchor: "#/discussion/#{discussion.id}")
```

Comment:
Is this supposed to start with a `#`? If it is, then the comment one should as well, right?

Also, can't this be generated later, when the user ask for it, on a layer that is actually responsible for urls, so that we don't need to go fetch them to the lost caves of `application.routes.url_helpers`? :mount_fuji: